lib/schemes.js

```diff
@@ -1,7 +1,8 @@
 'use strict'
 
-const { stringToHex } = require('./utils')
-const { scopedChars } = require('./scopedChars')
+const UUID_REG = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/
```

Comment:
Does this regexp cover all the UUID versions?
I was unable to find them in tests

Reply:
It does but indeed. Just the tests are not covering it.